Source/OrchBase/P_builder.cpp

```diff
@@ -9,6 +9,7 @@
 #include "P_device.h"
 #include "P_pintyp.h"
 #include "P_pin.h"
+#include "poets_msg.h" // TEMPORARY? include

```

Comment:
Why temporary? It seems quite useful to be able to know message type formats.

Reply:
I think this ties into your comment about moving `poets_msg.h` to a different location. I included this to get access to something, originally as a quick fix, but then never resolved it it seems.

Reply:
Removed the "temporary" comment.